flexmeasures/data/schemas/forecasting/pipeline.py

```diff
@@ -66,7 +66,12 @@ def validate_parameters(self, data: dict, **kwargs):
                     "start-predict-date must be before end-date",
                     field_name="start_predict_date",
                 )
-
+        elif not start_date:
+            raise ValidationError(
+                "Missing required option: either --start-date/--train-start or --from-date/--predict-start must be specified. "
```

Comment:
Doesn't `from-date` default to now, actually? In which case `--start-date` is not required, because it would default to 30 days before now, right?